### ISSUE TITLE
Docker mac bug

### ISSUE THREAD

**Issue by kyuta1** [docker/for-mac#578]
### Information

Diagnostic ID: 6D55ABC2-14A6-4200-A545-A32628EE0B4E
Docker for Mac: 1.12.0-a (Build 11213)
macOS: Version 10.12 (Build 16A313a)
[ERROR]  docker-cli
         docker ps failed
[OK]     app
[OK]     moby-syslog
[OK]     disk
[OK]     virtualization
[OK]     system
[OK]     menubar
[OK]     osxfs
[OK]     db
[OK]     slirp
[OK]     moby-console
[OK]     logs
[OK]     vmnetd
[OK]     env
[OK]     moby
[OK]     driver.amd64-linux
### Steps to reproduce
1. ...
2. ...

**Comment by mchiang0610** [comment]
Merging with #119

**Comment by djs55** [comment]
Docker for Mac beta 26 has improved support for macOS 10.12 Sierra: all the major show stopper issues which we are aware of have been fixed.

If you have the opportunity, please give it a try on the latest Sierra GM seed and open fresh issues for any problems you find.

- Release notes: https://download.docker.com/mac/beta/1.12.1.12048/NOTES
- Download: https://download.docker.com/mac/beta/Docker.dmg

Thanks for your help testing Docker for Mac with Sierra!

**Comment by samoht** [comment]
I am re-opening the issue as there is a weird error in the log:

```
Sep 15 19:50:35 Kyutas-MacBook-Pro Docker[com.docker.slirp][3652] <Error>: Lwt.async failure (Failure nth): Raised at file "pervasives.ml", line 32, characters 22-33
	Called from file "lib/dns_forward.ml", line 38, characters 41-77
	Called from file "lib/dns_forward.ml", line 102, characters 16-57
	Called from file "src/core/lwt.ml", line 795, characters 20-24
```

I have opened https://github.com/docker/vpnkit/issues/131 to keep track of that issue.

**Comment by djs55** [comment]
The `List.nth` failure should be fixed in both beta 30 and beta 30.1.

**Comment by yallop** [comment]
@kyuta1: I'm closing this issue, as we believe that the problem is fixed in recent betas.  Please reopen the issue if you see the problem again.  Thanks again for your report.
